components/ActivityForm/index.js

```diff
@@ -0,0 +1,129 @@
+import styled from "styled-components";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  margin: 15px;
+`;
+
+const StyledUlForm = styled.ul`
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  margin: 0px;
+  padding: 0px;
+`;
+const StyledListForm = styled.li`
+  list-style: none;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const StyledInputs = styled.input`
+  padding: 10px;
+  margin: 10px;
+`;
+
+const StyledTextarea = styled.textarea`
+  padding: 10px;
+  margin: 10px;
+`;
+export default function ActivityForm({ onSubmit }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formElements = new FormData(event.target);
+    const newActivity = Object.fromEntries(formElements);
+    newActivity.categoryIds = formElements.getAll("categoryIds");
+    const categoryIds = newActivity.categoryIds;
+    onSubmit(newActivity, categoryIds);
+  }
+
+  return (
+    <>
+      <StyledForm onSubmit={handleSubmit}>
+        <label htmlFor="title">Title: </label>
+        <StyledInputs
+          id="title"
+          name="title"
+          type="text"
+          placeholder="Please choose title.."
+          required
+        ></StyledInputs>
+        <label htmlFor="description">Description: </label>
+        <StyledTextarea
+          id="description"
+          name="description"
+          placeholder="Please choose description.."
+        ></StyledTextarea>
+        <button type="button">Please select categories</button>
```

Comment:
In the current implementation, the "Please select categories" element is a button with type="button". It appears to serve as a label for the checkboxes. Maybe using a different HTML element would be more appropriate